### PR TITLE
Schemas: Generate CRDs for core kinds

### PR DESCRIPTION
**Pull request by ryantxu**
This is extracted from #60207 -- it only generates CRDs without also trying to register them and set up a proxy.

### CODE REVIEW

File: pkg/codegen/jenny_crd_yaml.go [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+			return nil, err
+		}
+		if props.CRD.DummySchema {
+			ver.Schema = map[string]any{
+				"openAPIV3Schema": map[string]any{
+					"type": "object",
+					"properties": map[string]any{
+						"spec": map[string]any{
+							"type":                                 "object",
+							"x-kubernetes-preserve-unknown-fields": true,
```
Comment (ryantxu) [inline pkg/codegen/jenny_crd_yaml.go:90]: @toddtreece -- can you confirm this is the correct location?

Comment (toddtreece) [reply pkg/codegen/jenny_crd_yaml.go:90]: yep

File: pkg/kindsys/kindcats.cue [hunk 2]
```diff
@@ -100,6 +100,28 @@ Core: S=close({
 
 	lineage: { name: S.machineName }
 	lineageIsGroup: false
+
+	// crd contains properties specific to converting this kind to a Kubernetes CRD.
+	crd: {
```
Comment (toddtreece) [inline pkg/kindsys/kindcats.cue:105]: this seems to be missing the `description` added in https://github.com/grafana/grafana/pull/60207: https://github.com/grafana/grafana/blob/4567b502f23a03ef5ba69f642e4cf277b1255ca8/pkg/kindsys/kindcats.cue#L83-L88

File: pkg/services/dashboards/service/service.go [hunk 3]
```diff
@@ -16,10 +16,10 @@ func ProvideSimpleDashboardService(
 	svc *DashboardServiceImpl,
 	store entity.EntityStoreServer,
 	reg *corecrd.Registry,
-	bridge *bridge.Service,
+	troll *bridge.Service,
```
Comment (sdboyer) [inline pkg/services/dashboards/service/service.go]: ❤️

File: pkg/apimachinery/bridge/service.go [hunk 4]
```diff
@@ -145,19 +166,112 @@ func GenerateScheme(kinds []k8ssys.Kind) (*runtime.Scheme, error) {
...
+func (s *Service) createDashboard(reg *corecrd.Registry, clientSet *Clientset) {
+	namespace := "default"
+	// take the kindsys dashboard kind and alias it so it's easier to distinguish from dashboards.Dashboard
+	type dashboardKind = dashboard.Dashboard
+	// get the dashboard CRD from the CRD registry
+	dashboardCRD := reg.Dashboard()
+	// map native Grafana dashboard object to kindsys dashboard object
+
+	Uid := "ABCDIMAUID"
+	Title := "IMAK8SDASHIE"
```
Comment (sdboyer) [inline pkg/apimachinery/bridge/service.go]: ❤️ 👶🏻 ❤️

**Review by sdboyer — APPROVED** [review]
from my perspective, this is good - the details of the jennies are somewhat sloppy, but the high-level responsibility is established:

* make go CRD types
* make YAML CRD representation
* wrap them up into another registry layer for easy initialization and consumption

**Comment by sdboyer** [comment]
and yes, we'll absolutely reconcile these jennies with what's in app sdk

**Pull request merged**